lib/Parser/Parser.cpp

```diff
@@ -158,6 +160,24 @@ Token Lexer::getNextToken(std::string &ErrStr) {
     }
   }
 
+  // Parse the knownbits pattern followed by "var"
+  std::string IsKindNameVar = StringRef(Begin-4, 3);
```

Comment:
it seems like you are doing your own tokenization here, try to use the existing tokenizer instead